common/api/appui-react.api.md

```diff
@@ -4840,8 +4911,11 @@ export class UiFramework {
     // @internal (undocumented)
     static loggerCategory(obj: any): string;
     static readonly onUiVisibilityChanged: UiVisibilityChangedEvent;
+    static openContextMenu(items: AbstractMenuItemProps[], location: XAndY, anchorElement?: HTMLElement): boolean;
```

Comment:
Same concern as before, will we deprecate these abstract types? If yes, we need a better strategy for these new APIs/types...

Reply:
That's not a good option, since we are already introducing a `@public` type which we can't change after? Maybe we could mark these as `@internal` for now and have no replacement for now (not ideal either) :?

Reply:
Right, I was thinking it was core and could sit on master while devs worked off release branches... 
Hm maybe I'll suck it up and try to fit this in here :) Hopefully won't be too messy.

Reply:
Taking a look at the remaining appui-abstract types I've used in new APIs in UIFramework... some optimism I can find a way forward.

Reply:
Alright, as discussed, any `UiFramework` methods added which utilize appui-abstract functionality that doesn't have a deprecation plan are `@internal` and corresponding `FrameworkUiAdmin` methods have not deprecated. Similarly, `FrameworkUiAdmin` itself will not yet be deprecated. Let me know if you see anything I missed.